dbms/src/Flash/Executor/ResultQueue.h

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <Common/MPMCQueue.h>
+#include <Core/Block.h>
+
+#include <mutex>
+
+namespace DB
+{
+class ResultQueue
```

Comment:
I’m wondering why we need a new queue

Reply:
ok, new queue removed and use `using ResultQueue = MPMCQueue<Block>` directly.